contracts/lib/ZoneInteraction.sol

```diff
@@ -211,6 +219,15 @@ contract ZoneInteraction is ZoneInteractionErrors, LowLevelHelpers {
         }
     }
 
+    /**
+    * @dev Calls the specified target with the given data and checks the status of the call.
```

Comment:
Should add some additional context here, namely that revert reasons will be "bubbled up" if one is returned, otherwise reverting calls will throw a generic error based on the supplied error handler